tests/integration/motions/test_viewset.py

```diff
@@ -617,8 +617,8 @@ def setUp(self):
         self.client.login(username='admin', password='admin')
 
         self.admin = get_user_model().objects.get()
-        self.group_in = self.admin.groups.first()
```

Comment:
@normanjaeckel Just FYI: `self.admin.groups.first()` has the type `django.contrib.auth.models.Group` and not our own group model. This seems to be broken..
Now, you cannot assign groups from users to comments..